guac-SBOM-ingestion.md

```diff
@@ -0,0 +1,88 @@
+## SBOM Ingestion in GUAC
+
+### Overview
+
+Software Bill of Materials (SBOM) ingestion is essential in GUAC (Graph for Understanding Artifact Composition) to help track and analyze dependencies, vulnerabilities, and software supply chain metadata. This documentation explains how to ingest SBOMs in GUAC, covering:
+- **Supported formats and limitations**
+- **Ingestion methods**
+- **Daemon-mode ingestion**
+- **Manual vs. automatic ingestion** 
+
+---
+
+### Supported SBOM Formats
+
+GUAC supports a variety of SBOM formats, making it compatible with several tools and standards:
+- **SPDX**: A widely used open standard for software package metadata.
+- **CycloneDX**: An SBOM specification built for security use cases.
+- **Syft**: A tool generating SBOMs directly from container images and filesystems.
+
+**What is not supported**:
+- Proprietary SBOM formats that deviate from open standards.
+- Incomplete SBOMs without metadata on dependencies or vulnerabilities might have limited utility for ingestion.
+
+---
+
+### Ingestion Methods
+
+You can ingest SBOMs in GUAC using several approaches, each tailored to different needs:
+
+1. **Manual Ingestion**:
+   - Use the GUAC CLI to manually push SBOMs to the backend.
+   - Example:
+     ```bash
+     guac-cli ingest --file my-sbom.spdx.json
```

Comment:
Where did this come from? `guac-cli` isn't a binary we produce. A correct example would be `guacone collect files my-sbom.spdx.json`